tests/serde-se.rs

```diff
@@ -285,28 +285,41 @@ mod with_root {
         => "<root>true</root>");
     serialize_as!(tuple:
         (42.0, "answer")
-        => "<root>42</root><root>answer</root>");
+        => "<root>42</root>\
+            <root>answer</root>");
     serialize_as!(tuple_struct:
         Tuple(42.0, "answer")
-        => "<root>42</root><root>answer</root>");
+        => "<root>42</root>\
+            <root>answer</root>");
     serialize_as!(struct_:
         Struct {
             float: 42.0,
             string: "answer"
         }
-        => r#"<root float="42" string="answer"/>"#);
+        => "<root>\
+                <float>42</float>\
+                <string>answer</string>\
+            </root>");
     serialize_as!(nested_struct:
         NestedStruct {
             nested: Nested { float: 42.0 },
             string: "answer",
         }
-        => r#"<root string="answer"><nested float="42"/></root>"#);
+        => "<root>\
+                <nested>\
+                    <float>42</float>\
+                </nested>\
+                <string>answer</string>\
+            </root>");
     serialize_as!(flatten_struct:
         FlattenStruct {
             nested: Nested { float: 42.0 },
             string: "answer",
         }
-        => r#"<root><float>42</float><string>answer</string></root>"#);
+        => "<root>\
+                <float>42</float>\
+                <string>answer</string>\
```

Comment:
What is the mechanism for having a different expectation of this test vs. the previous one when the structs are declared the same?